qaframework-bdd-tests/src/test/resources/features/refapp-2.x/15-forms/forms.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Form Management
+
+  Background:
+  Given  a user click on manage html forms
+
+  @selenium
+  @forms
```

Comment:
isn't the tag supposed to be `@form`?

Reply:
There are more than one form in an encounter, Thats the ideal point here.